xapian-core/tests/harness/testsuite.cc

```diff
@@ -706,9 +711,20 @@ test_driver::do_run_tests(vector<string>::const_iterator b,
 		case PASS:
 		    ++res.succeeded;
 		    if (verbose || !use_cr) {
-			out << col_green << " ok" << col_reset << endl;
```

Comment:
You could just keep everything to `col_reset`, then have the "long run" conditional do `col_yellow` to `col_reset` again (and no else leg), with a final `\n` at the end. I _think_ that would be a little more readable, but I could be wrong.

Reply:
changed accordingly.